src/PowerShellEditorServices/Language/CommandHelpers.cs

```diff
@@ -14,6 +16,20 @@ namespace Microsoft.PowerShell.EditorServices
     /// </summary>
     public class CommandHelpers
     {
+        private static HashSet<string> NounBlackList =
+            new HashSet<string>
+            {
+                "Module",
```

Comment:
Probably not worth the extra effort at the moment, I imagine many people won't even notice it.  If people do notice and complain, I'll definitely find a way to make it a bit more obvious what's going on.